modules/govuk/manifests/apps/publicapi.pp

```diff
@@ -15,12 +16,15 @@
 
     $whitehallapi = "whitehall-frontend.${app_domain_internal}"
     $rummager_api = "search.${app_domain_internal}"
-    # Content Store is always public
-    $content_store_api = "content-store.${app_domain}"
+    $content_store_api = "content-store.${app_domain_internal}"
```

Comment:
The content store is not "always public" anymore then.

Reply:
Oh, wait, now it matches the others in this file (whitehall and rummager), so it's OK.